cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/cmd/BuildCommand.java

```diff
@@ -189,7 +191,7 @@ public void execute() {
             // "ballerina.conf" in the source root path is taken.
             LauncherUtils.loadConfigurations(sourceRootPath, configFilePath);
 
-            if (jvmTarget) {
+            if (jvmTarget || JVM_TARGET.equals(System.getProperty(BALLERINA_TARGET))) {
```

Comment:
Need to keep jvmTarget option since we have some tool-integration test on running a jar which by default use ballerina-tools distribution. We can do some modifications in integration test utils to run jvm integration tests on jballerina distribution. hence keep this for moment.